src/memory.rs

```diff
@@ -21,6 +21,7 @@ declare_log_buffer!(name = ERROR, capacity = 1000);
 thread_local! {
     // Transient static data: this is reset when the canister is upgraded.
     pub static TRANSIENT_METRICS: RefCell<Metrics> = RefCell::new(Metrics::default());
+    pub static TRANSIENT_SUBNET_SIZE: RefCell<u32> = RefCell::new(DEFAULT_NODES_IN_SUBNET);
```

Comment:
So, it's called "transient" because this information is not persisted?
It's a bit misleading in my opinion because an upgrade doesn't change the subnet size.

Reply:
Do you know if there is a conventional name for non-stable memory on the IC? I've seen "main memory" and "flexible memory" but neither seem to work better in this case. I suppose we could remove the `TRANSIENT_` prefix and define these constants in `stable::*` and `unstable::*` modules, or something along those lines.

Reply:
I like the approach with different modules better. "Unstable subnet size" also sounds weird.   :slightly_smiling_face: 

But we can go with your suggestion for now and have some bike-shedding fun later.